.github/workflows/tests.yaml

```diff
@@ -79,10 +83,28 @@ jobs:
         uses: abatilo/actions-poetry@v3
         with:
           poetry-version: "1.3.2"
+      - name: Get GitHub-Actions
+        run: |
+          git clone https://github.com/QuTech-Delft/GitHub-Actions.git
+          cd GitHub-Actions
+          git switch feature/QI2-1222/include-auth-script
+          pwd
+      - name: Get authentication token
+        run: |
+          python -m venv env
+          . ./env/bin/activate
+          mkdir -p ~/.quantuminspire
+          pip install -r requirements.txt
+          python get_oauth_tokens.py > ~/.quantuminspire/config.json
+          cat ~/.quantuminspire/config.json
+          deactivate
+        working-directory: GitHub-Actions/scripts/oauth_tokens
       - name: Install packages
         run: poetry install
       - name: Run e2e test
-        run: poetry run python tests/e2e_test.py "${{ matrix.backend-name }}"
+        run: |
+          cat ~/.quantuminspire/config.json
```

Comment:
No need to do it here again if we're doing it in the previous step already WDYT?